agent/agent/src/main/java/com/microsoft/applicationinsights/agent/StartupProfiler.java

```diff
@@ -117,7 +117,9 @@ public void run() {
     }
 
     private void write(ThreadInfo threadInfo) {
-      if (capture(threadInfo)) {}
+      if (capture(threadInfo)) {
+        // TODO look like this is not used.
+      }
```

Comment:
it's ok to remove this and the `capture` method below, that was just to optionally filter down the amount of threads dumped, but I don't think worth adding a setting for (just want to keep this simple for customers, and we can filter down as needed)
```suggestion
```